content/articles/data-access-with-ado.net-in-micro-focus-visual-cobol/index.md

```diff
@@ -0,0 +1,144 @@
+### Introduction
+`ADO` is a Microsoft technology that stands for `ActiveX Data Object`. It is a collection of ActiveX controls that give you programmatic access to Microsoft's most recent data access technologies. As the name suggests, it's a wrapper for fundamental database access techniques.
+
+`ADO.NET` is the next step in data access technology advancement. Using the previously mentioned ADO technology, `ADO.NET` extends this by adding XML into a common model that applies to both relational and text-based XML data. First-class languages are supported by the .NET Framework's .NET API, which is managed code. .NET can use the classes that have already been established, rather than having to reinvent the wheel when it comes to data access. It's essentially a set of access and manipulation processes that a developer can utilize to gain access to his or her data. The best part is that you don't need to install anything because it's already included in the .NET Framework.
+
+### Table of contents:
+- [Basic ADO Flow](#basic-ado-flow)
+- [Preparation](#preparation)
+- [Create a Connection](#create-a-connection)
+- [Create a Command](#create-a-command)
+- [Read the Data](#read-the-data)
+- [Housekeeping](#housekeeping)
+- [Other Data Providers](#other-data-providers)
+- [Wrap-Up](#wrap-up)
+
+### Basic ADO Flow
+What do I do now that Microsoft has provided me with this one-of-a-kind library of data access procedures? How can I get started? What do I look for when I am looking for a place to stay? We will start by looking at `SQL Server's namespace`, which governs data access. At the end of this article, we will go over different data providers, but for now, we will stick with SQL Server. System.Data.SqlClient is the namespace to remember if you want to connect to SQL Server.
+
+> Keep this namespace in mind. It's critical!
+
+There are presently 16 classes in the SQLClient namespace. These classes provide a programmatic way to establish connections to databases, verify permissions, manage exceptions, and do other duties related to data access. Let's get started using ADO.
+
+To view data object, follow this basic procedure:
+1. Establish a link to the data object.
+2. Make a command that the data object will follow.
+3. Consider the data that the data object has provided.
+
+To complete the actions listed above, we will need to employ the following classes:
+1. System.Data.SqlClient.SqlConnection
+2. System.Data.SqlClient.SqlCommand
+3. System.Data.SqlClient.SqlDataReader
+
+### Preparation

```

Comment:
What is the goal of this code tutorial? State the objectives. State the expected end result.